Demo/Common/IntegrationExampleDropIn.swift

```diff
@@ -52,6 +52,8 @@ extension IntegrationExample {
         configuration.actionComponent.threeDS.requestorAppURL = URL(string: ConfigurationConstants.returnUrl)
         configuration.card.billingAddress.mode = .postalCode
         configuration.paymentMethodsList.allowDisablingStoredPaymentMethods = true
+        configuration.cashAppPay = CashAppPayConfiguration(redirectURL: URL(string: "ui-host://callback")!,
```

Comment:
Maybe we should have new constant in `ConfigurationConstants` ?

Reply:
Oh nice one, we already have it. Will use `URL(string: ConfigurationConstants.returnUrl)`